functions/go/apply-setters/README.md

```diff
@@ -106,7 +106,7 @@ $ kpt fn eval --image gcr.io/kpt-fn/apply-setters:unstable --fn-config ./apply-s
 Alternatively, setter values can be passed as key-value pairs in the CLI
 
 ```shell
-$ kpt fn eval --image gcr.io/kpt-fn/apply-setters:unstable -- image=ubuntu replicas=3
+$ kpt fn eval --image gcr.io/kpt-fn/apply-setters:unstable -- tag=1.16.2 replicas=3
```

Comment:
Thank you for finding this! It looks like `replicas` should also be renamed to `nginx-replicas`

Reply:
Yup, I missed that one.  Just added to the PR.